examples/tools/crosshairs.md

```diff
@@ -1,6 +1,38 @@
 ---
-layout: needs-example
+layout: tool-simple
 title: Crosshairs Tool
 toolName: Crosshairs
 toolType: stack
 ---
+
+
+<h2 class="title is-2">How to set up the {{page.toolName}} tool:</h2>
+
+{% highlight javascript %}
+// Init cornerstone tools
+cornerstoneTools.init()
+
+// Enable any elements, and display images
+const imageIds = [`image_id_1`,`image_id_2`,]
+// ...
+
+// Add our tool, and set it's mode
+const {{page.toolName}}Tool = cornerstoneTools.{{page.toolName}}Tool
```

Comment:
Very cool to see that you've picked up on page/template variables :+1:

Because this is the same page that sets that variable, please don't feel like you have to use them here. They provide more benefit in the reusable templates where the values are inherently more dynamic.

No right/wrong answer here, just providing thoughts ^_^

Reply:
Yeah, I think there are no benefits using those variables this way. Rolling back to the hardcoded name :)